### DISCUSSION TITLE
How to use Interpret & compilation mode?

### DISCUSSION THREAD

**Discussion by lamualfa** [windicss/vite-plugin-windicss#219] [Q&A] [answered]
I saw this [page](https://windicss.org/posts/modes.html) in [windicss.org](windicss.org). and how i can implent it in my `react-ts` project? does `vite-plugin-windicss` support it?

**Comment by antfu** [comment, marked as answer]
Sorry, currently vite plugin does not support compilation mode.

**Reply by jinzhubaofu** [reply]
@antfu Hi, do we have any plan to support compilation mode in vite-plugin-windicss?

**Reply by JenuelDev** [reply]
ayn update on this?
